x-pack/plugins/observability/public/components/shared/exploratory_view/exploratory_view.tsx

```diff
@@ -129,7 +133,11 @@ export function ExploratoryView({
                         <LastUpdated chartTimeRange={chartTimeRangeContext} />
                       </EuiFlexItem>
                       <EuiFlexItem grow={false}>
-                        <EuiButton iconType="refresh" onClick={() => setLastRefresh(Date.now())}>
+                        <EuiButton
+                          iconType="refresh"
+                          onClick={() => setLastRefresh(Date.now())}
+                          size="s"
```

Comment:
We shouldn't change the refresh button size, but instead move the date picker to the page header content instead, like with the Observability overview. Then we avoid making the buttons inconsistent with other instances and use the abilities section in the page header component as suggested in the guidelines.

<img width="1498" alt="CleanShot 2022-04-20 at 13 26 45@2x" src="https://user-images.githubusercontent.com/4104278/164220696-088e3f31-2823-476d-8cfd-bd1f07410ca6.png">

Reply:
Yes, i will do a follow up to move the button up and also fix it, it actually doesn't work at the moment.

Reply:
@shahzad31 OK, thanks - that sounds good